vm/devices/net/netvsp/src/lib.rs

```diff
@@ -331,29 +331,50 @@ impl RxBufferRange {
     fn send_if_remote(&self, id: u32) -> bool {
         if self.id_range.contains(&id) {
             false
+        } else if id < RX_RESERVED_CONTROL_BUFFERS
+            && self.remote_ranges.excluded_primary_send.is_some()
+        {
+            self.remote_ranges
```

Comment:
I don't yet understand this case. Why wouldn't we just assign the reserved region to queue 0 so that this path never hits? Let's see.

Reply:
Oh, I see. I am not certain this case hits in practice (a non-primary queue shouldn't be completing a remote queue's IDs; this only happens across RSS reconfiguration, where IDs can move between queues, but the reserved buffers will never move off of queue 0). But I suppose it's not unreasonable to support this anyway, since we do seem to support it in the existing path.

Reply:
I think an easier-to-understand approach might be to have a `Sender` specifically for the reserved buffers, always there, rather than just when we're in this mode. `Sender` (and `UnboundedSender` for release/2411) implement an efficient `Clone`, so this can just be the same as queue 0's existing sender for the case where queue 0 is active.

So basically, rename `excluded_primary_send` to just `control_buffer_send` or whatever, and remove the `Option`.